internal/controller/bucket_controller.go

```diff
@@ -110,7 +110,12 @@ func (r BucketReconciler) reconcileNormal(ctx context.Context, objectStorageServ
 			return ctrl.Result{}, errors.WithStack(err)
 		}
 	} else {
-		logger.Info("Bucket exists and you already own it.")
+		logger.Info("Bucket exists and you already own it, let's update it")
+		err = objectStorageService.UpdateBucket(ctx, bucket)
```

Comment:
Unless we want to delete all our storage accounts, we need to be able to update azure storage